workflows/python/sdk/order-processor/workflow_activities.py

```diff
@@ -0,0 +1,101 @@
+# , when_any
+from model import InventoryItem, Notification, InventoryRequest, OrderPayload, OrderResult, PaymentRequest, InventoryResult
+from dapr.clients import DaprClient
+from util import get_address
+
+store_name = "statestore-actors"
+
+address = get_address()
+
+def order_processing_workflow(ctx: DaprWorkflowContext, orderPayload: OrderPayload):
+    logging.basicConfig(level=logging.INFO)
+    order_id = ctx.instance_id
+    yield ctx.call_activity(notify_activity, input=Notification(message=f'Received order {order_id} for {orderPayload.quantity} {orderPayload.item_name} at ${orderPayload.total_cost} !'))
+    result = yield ctx.call_activity(verify_inventory_activity, input=InventoryRequest(request_id=order_id, item_name=orderPayload.item_name, quantity=orderPayload.quantity))
+    if not result.success:
+        yield ctx.call_activity(notify_activity, input=Notification(message=f'Insufficient inventory for {orderPayload.item_name}!'))
+        return OrderResult(processed=False)
+    
+    if orderPayload.total_cost > 50000:
+        yield ctx.call_activity(requst_approval_activity, input=orderPayload)
+        approval_flag = ctx.wait_for_external_event("manager_approval")
+        # Temporarily disabling timeout, as when_any PR is not merged in python-sdk
+        # timeout_event = ctx.create_timer(timedelta(seconds=200))
+        # winner = yield when_any([approval_flag, timeout_event])
+        # if winner == timeout_event:
+        #     yield ctx.call_activity(notify_activity, input=Notification(message=f'Payment for order {order_id} has been cancelled due to timeout!'))
+        #     return OrderResult(processed=False)
+        approval_result = yield approval_flag
+        if approval_result["approval"]:
+            yield ctx.call_activity(notify_activity, input=Notification(message=f'Payment for order {order_id} has been approved!'))
+        else:
+            yield ctx.call_activity(notify_activity, input=Notification(message=f'Payment for order {order_id} has been rejected!'))
+            return OrderResult(processed=False)    
+    
+    yield ctx.call_activity(process_payment_activity, input=PaymentRequest(request_id=order_id, item_being_purchased=orderPayload.item_name, amount=orderPayload.total_cost, quantity=orderPayload.quantity))
+
+    try:
+        yield ctx.call_activity(update_inventory_activity, input=PaymentRequest(request_id=order_id, item_being_purchased=orderPayload.item_name, amount=orderPayload.total_cost, quantity=orderPayload.quantity))
+    except Exception:
+        yield ctx.call_activity(notify_activity, input=Notification(message=f'Order {order_id} Failed!'))
+        return OrderResult(processed=False)
+
+    yield ctx.call_activity(notify_activity, input=Notification(message=f'Order {order_id} has completed!'))
+    return OrderResult(processed=True)
+
+def notify_activity(ctx: WorkflowActivityContext, input: Notification):
+    # Create a logger
```

Comment:
As you suggested in another comment, let's add documentation strings to each of these functions to provide more context on what they are and what they are for.

Reply:
Have added.